.github/workflows/k8s.yml

```diff
@@ -0,0 +1,32 @@
+name: Kubernetes
```

Comment:
Could you add the ASF license header? It looks like we don't have the RAT checks set up in CI yet.

Reply:
I would also be okay with explicitly excluding GitHub workflows from RAT checks if we had RAT set up, so maybe this is fine.

Reply:
I'll go ahead and merge so we can test. I added a bullet to https://github.com/apache/datafusion-ray/issues/2 for setting up RAT.